packages/nextjs/pages/index.tsx

```diff
@@ -262,12 +276,20 @@ const Home: NextPage = () => {
                         value={localContractAbi}
                         onChange={e => setLocalContractAbi(e.target.value)}
                       ></textarea>
-                      <button
-                        className="btn btn-primary min-h-fit h-10 px-4 mb-12 text-base font-semibold border-2 hover:bg-neutral hover:text-primary"
-                        onClick={handleUserProvidedAbi}
-                      >
-                        Import ABI
-                      </button>
+                      <div className="flex gap-2">
+                        <button
+                          className="btn btn-primary min-h-fit h-10 px-4 mb-2 text-base font-semibold border-2 hover:bg-neutral hover:text-primary"
+                          onClick={handleUserProvidedAbi}
+                        >
+                          Import ABI
+                        </button>
+                        <button
+                          className="btn btn-secondary btn-ghost min-h-fit h-10 px-4 mb-12 text-base font-semibold border-2 hover:bg-neutral hover:text-primary"
+                          onClick={loadStoredAbi}
+                        >
+                          Load stored ABI
+                        </button>
```

Comment:
Maybe just show the button if there is a stored ABI for the contract? So we don't get this message:

![image](https://github.com/BuidlGuidl/abi.ninja/assets/55535804/f2a5ec1c-642f-43d0-9b00-318059bfffb9)

Also not sure about the UI for the button. Should it have primary style since it's maybe the usual option if you have the ABI stored? If so, should we change "Import ABI" button where there is ABI in local storage? 
Just thinking out loud,  I really don't know, but something feels not right like that to me

Reply:
Do you mean not rendering that button if we don't have an ABI?

I think there is much room for improvement here.

Maybe we should make the ui larger, or migrate it to shadcn while doing that.

Reply:
> Do you mean not rendering that button if we don't have an ABI?

Yeah! I'm thinking we could have a workflow like this:

![image](https://github.com/BuidlGuidl/abi.ninja/assets/55535804/bc4ad69e-d54e-4fab-ad9b-308581b0a253)

If exists stored ABI, the main option to show is "Load stored ABI", but we give the user the possibility to dismiss and import new ABI.

> Maybe we should make the ui larger, or migrate it to shadcn while doing that.

That feels a bit overkill to me right now! Maybe if we migrate SE-2 to shadcn or there is a specific need on Abi ninja I agree on that, but feels not needed to me atm.